account_banking_sepa_credit_transfer/data/account_payment_method.xml

```diff
@@ -4,7 +4,7 @@
         <field name="name">SEPA Credit Transfer to suppliers</field>
         <field name="code">sepa_credit_transfer</field>
         <field name="payment_type">outbound</field>
-        <field name="bank_account_required" eval="True" />
+        <field name="bank_account_required" eval="False" />
         <field name="pain_version">pain.001.001.03</field>
```

Comment:
Why do you change this?? You are modifying the file for credit transfers !

Reply:
Such bank account is as well not needed, as it's auto-populated on payment order filling.

Reply:
Then change the code, don't change the value of this field.

Reply:
Or remove the field. It would be ridiculous to change the value of those field to False.

Reply:
Why is ridiculous? We just don't want that option enabled for this payment method by default.

Reply:
I changed the value in order to make it clear that it is not required

Reply:
This is completely ridiculous!!! In the definition of the account.payment.method for credit transfer, we say that bank account is not required !!! If it's not required for credit transfer, for which payment method will it be required? You should have changed the code to make it required starting from the payment line instead of starting from the invoice. That could make sense. But the change here doesn't make sense at all. You're screwing up the architecture.

Reply:
No, it isn't. Requiring it at invoice level is what it doesn't make sense, and more populate it when selecting the partner. Maybe the invoice was done 8 months ago, and the partner has changed 3 times of bank account. The bank account is populated when you do the payment/debit order, and shouldn't be before if you want to avoid surprises. If you tell me, I would totally remove such checks.

Reply:
OK, thanks for giving the real reason for the change... at last ! I can agree with this reason, but it's not written in the title of the PR nor in the first comment of @etobella You should have told your motivations clearly, I would have advised on the proper solution. I'm the author of this code, so I have a good idea of the consequences, and it would have avoided the breakage.